.buildkite/pipeline.yml

```diff
@@ -107,3 +107,27 @@ steps:
         depends_on:
           - "operator-image-build"
           - "e2e-tests-image-build"
+
+  - group: helm-release
+
+    steps:
+
+      - label: ":helm: release eck-operator helm chart - pipeline upload"
+        if: build.tag != null
+        commands:
+          - buildkite-agent pipeline upload .buildkite/pipeline-helm-release-eck-operator.yml
+
+      - label: ":helm: release eck-operator helm chart manually - pipeline upload"
+        if: build.message == "release eck-operator helm chart"
+        commands:
+          - buildkite-agent pipeline upload .buildkite/pipeline-helm-release-eck-operator.yml
+
+      - label: ":helm: release eck-resources helm charts manually - pipeline upload"
+        if: build.message == "release eck-resources helm charts"
+        commands:
+          - buildkite-agent pipeline upload .buildkite/pipeline-helm-release-eck-resources.yml
+
+      - label: ":helm: release all helm charts manually - pipeline upload"
+        if: build.message == "release all helm charts"
+        commands:
+          - buildkite-agent pipeline upload .buildkite/pipeline-helm-release-all.yml
```

Comment:
I think we said we will publish daily snapshots of all helm charts daily from `main` to the `dev` repository. There will be more change on this part when we introduce the pipeline-release.yml definition so I'm ok with merging the PR in this state and moving forward.